source/PlayerInfo.cpp

```diff
@@ -1043,6 +1054,22 @@ void PlayerInfo::Land(UI *ui)
 			ui->Push(new Dialog(message));
 	}
 	
+	// Hire crew for the flagship if the user changed it from the default.
+	if(Preferences::Has("Rehire flagship crew on landing") && hasSpaceport && flagshipCrew)
```

Comment:
Maybe "Rehire extra crew when lost" would make more sense? (Just so it's clear that this is only talking about *extra* crew, and that normal crew are going to be replenished no matter what.)